InternProject/DAL/Models/DeliveryTrack/DCBookingDetailModel.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using InternProject.DAL.Models.OrderTrack;
+using InternProject.DAL.Models.ShipmentTrack;
+
+namespace InternProject.DAL.Models.DeliveryTrack
+{
+    public class DCBookingDetailModel
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [StringLength(30)]
+        public string Line { get; set; }
+
+        public int Quantity { get; set; }
+
+        public int Cartons { get; set; }
+
+        public decimal Cube { get; set; }
+
+        [StringLength(50)]
+        public string Item { get; set; }
+
+        [ForeignKey("Order")]
+        public int OrderId { get; set; }
+
+        //[ForeignKey("ManifestModel")]
+        public string Container { get; set; }
+
+        [ForeignKey("DCBookingModel")]
```

Comment:
For the default behavior of EF, DCBookingId and DCBooking property are naming convention which will automatically  setup a foreign key

Don't need to specify [ForeignKey] here, after you change to DCBooking below

Reply:
commented out unecessary [ForeignKey] annotation